STATUS: CLEANER DOES NOT FIX
MODIFIED: N/A
MODIFIED FILE: N/A
CLEANER: io.github.dddplus.runtime.registry.IntegrationTest.extThrowException
CLEANER FILE: /home/lichengpeng/test/automated_script/funkygao/cp-ddd-framework/dddplus-test/src/test/java/io/github/dddplus/runtime/registry/IntegrationTest.java
POLLUTER: io.github.dddplus.testing.LogAssertTest.contains
POLLUTER FILE: /home/lichengpeng/test/automated_script/funkygao/cp-ddd-framework/dddplus-test/src/test/java/io/github/dddplus/testing/LogAssertTest.java
ORIGINAL CLEANER SIZE: 5
NEW CLEANER SIZE: 5
ELAPSED TIME: 0.0
==========================
@@ -0,0 +0,19 @@
+ {
+     {
+         fooModel = new FooModel();
+         fooModel.setPartnerCode(FooPartner.CODE);
+         fooModel.setB2c(true);
+         // ExtensionInvocationHandler.extInvokeTimerExecutor的线程池缩小到10，方便并发测试
+         System.setProperty("invokeExtMaxPoolSize", "10");
+     }
+     try {
+         // B2BExt
+         fooModel.setPartnerCode("");
+         fooModel.setB2c(false);
+         fooDomainService.submitOrder(fooModel);
+     } catch (Throwable ex) {
+     } finally {
+     }
+     {
+     }
+ }
